src/styles/blocks/content.scss

```diff
@@ -0,0 +1,16 @@
+.content {
+  display: grid;
+  justify-content: center;
+
+  &__container {
+    width: 944px;
+    height: 862px;
+    padding: 50px 40px 72px;
+
+    display: grid;
+    grid-template-columns: repeat(4, 198px);
+    row-gap: 47px;
+    column-gap: 49px;
```

Comment:
Pay attention that the gaps should be 46 and 48px. Probably that's the reason of failing tests
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/b995bb0c-a054-403f-8242-5251e6b479a4)